gensim/scripts/make_wiki.py

```diff
@@ -1 +0,0 @@
-make_wikicorpus.py
```

Comment:
When there's a conflict, git clearly shows you which parts are in conflict: it puts the pieces of code between `>>>>>` and `<<<<<<` markers.

I'm not sure what happened here or why you replaced the symlinks, but we definitely want to keep the original symlinks (and not full files, as in this PR).

This is also connected to debugging -- the fewer the changes, the smaller the surface where the bug may have been introduced. The segfault will be easier to debug with a smaller PR.